docs/developer/app-store/apps/cms.mdx

```diff
@@ -0,0 +1,188 @@
+
+To execute the initial sync, you must have at least one connection configured. You can then use it to scan the entire Saleor database for products and send them to the provider.
+
+:::caution Limitation
+At the moment, the App performs bulk sync on the front end. That means when this is happening, the App must stay open. Closing the window will stop the operation.
+:::
+
+## Configuration
+
+To configure the app, follow the instructions built into the app.
+
+Usually, you will need to perform the following steps:
+
+1. Create a provider configuration for the chosen provider.
+2. Assign the created configuration to a selected channel.
+3. Perform an initial sync.
+4. Edit or create some product variant to see if it is synchronized with the provider.
+
+### CMS fields setup
+
+The App allows you to map Saleor product fields to the CMS fields.
+Some providers allow the App to fetch and display the available fields in an autocomplete. If autocomplete is unavailable, you must ensure the field names are correct.
+
+### Provider configuration
+
+Each of the CMS platforms has slight differences in configuration and behavior due to their APIs and limitations.
+
+#### Contentful
+
+- Fields are autocompleted after you provide [space ID](https://www.contentful.com/help/find-space-id/) and a [token](https://www.contentful.com/help/personal-access-tokens/).
+- Entries will be created with an ID that matches Saleor product variant ID.
+- Bulk sync operations are limited to 2 items per second due to Contentful API limitations.
+- All fields must be the type of `string/text`, except `channels`, which should be type `json`.
+
+#### [DatoCMS](https://www.datocms.com/)
+
+- Fields are autocompleted after you provide a full-access [API token (Content Management API)](https://www.datocms.com/docs/content-management-api/authentication).
+- Entries will be created with IDs generated by DatoCMS. You should set the variant ID field to be unique.
+- Bulk sync operations will be rate-limited by DatoCMS client itself.
+- All fields must be the type of `string/text`, except `channels`, which should be type `json`.
+
+#### [Strapi](https://strapi.io/)
+
+- Fields are not autocompleted. You must provide an auth token with full access to the product variant entity and API base URL (without `/api` suffix).
+- Item type must be plural, eg, `saleor-products` instead `saleor-product`.
+- All fields must be the type of `string/text`, except `channels`, which should be type `json`.
+
+#### [Builder.io](https://www.builder.io/)
+
+- Fields are not autocompleted. You must ensure mapping is correct.
+- Builder.io doesn't support unique constraints on fields - hence the app will not guarantee that duplicates will not be created.
+
+## General limitations and known issues
+
+- App is unidirectional. Changes in CMS will not be reflected in Saleor.
```

Comment:
**question**: We have the same point in the "Assumptions" section. Is that expected?